doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -80,81 +80,97 @@
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m 2022-08-02 17:34:19,313\tWARNING __init__.py:192 -- DeprecationWarning: `ray.worker.get_resource_ids` is a private attribute and access will be removed in a future Ray version.\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/ray/python/ray/_private/workers/default_worker.py\", line 237, in <module>\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     ray._private.worker.global_worker.main_loop()\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/ray/python/ray/_private/worker.py\", line 754, in main_loop\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     self.core_worker.run_task_loop()\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/ray/python/ray/_private/function_manager.py\", line 674, in actor_method_executor\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     return method(__ray_actor, *args, **kwargs)\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/ray/python/ray/util/tracing/tracing_helper.py\", line 466, in _resume_span\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     return method(self, *_args, **_kwargs)\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/mambaforge/envs/ray/lib/python3.8/site-packages/xgboost_ray/main.py\", line 474, in __init__\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     _set_omp_num_threads()\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/mambaforge/envs/ray/lib/python3.8/site-packages/xgboost_ray/main.py\", line 280, in _set_omp_num_threads\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     ray_cpus = _ray_get_actor_cpus()\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/mambaforge/envs/ray/lib/python3.8/site-packages/xgboost_ray/main.py\", line 262, in _ray_get_actor_cpus\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     resource_ids = ray.worker.get_resource_ids()\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m   File \"/Users/cindyz/ray/python/ray/__init__.py\", line 196, in __getattr__\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m     traceback.print_stack()\n",
+      "\u001b[2m\u001b[36m(XGBoostTrainer pid=50560)\u001b[0m 2022-08-02 17:34:19,445\tINFO main.py:1025 -- [RayXGBoost] Starting XGBoost training.\n",
+      "\u001b[2m\u001b[36m(_RemoteRayXGBoostActor pid=50569)\u001b[0m [17:34:19] task [xgboost.ray]:5182388976 got new rank 0\n"
```

Comment:
Running the first cell for this python notebook on [master](https://github.com/ray-project/ray/commit/8b7e4ac70135d3f11398236d8e4e2e77950f22af) ([link](https://docs.ray.io/en/master/ray-air/examples/serving_guide.html) to doc) prints this traceback.